packages/api/lib/writeRecords/write-execution.js

```diff
@@ -88,87 +82,49 @@ const buildExecutionRecord = ({
-    writeConstraints = true,
-  } = params;
-  return await upsertExecution({
-    esClient,
-    updates: apiRecord,
-    index: process.env.ES_INDEX,
-  }, writeConstraints);
-};
-
 /**
  * Write execution record to databases
  *
- * @param {Object} params
- * @param {Object} params.apiRecord - Execution API record to be written
- * @param {Object} params.postgresRecord - Execution PostgreSQL record to be written
- * @param {Object} params.knex - Knex client
- * @param {Object} [params.executionPgModel] - PostgreSQL execution model
- * @param {number} [params.updatedAt] - updatedAt timestamp to use when writing records
- * @param {Object} [params.esClient] - Elasticsearch client
- * @returns {Promise<Object>} - PostgreSQL execution record that was written to the database
+ * @param {object} params
+ * @param {object} params.postgresRecord - Execution PostgreSQL record to be written
+ * @param {object} params.knex - Knex client
+ * @param {object} [params.executionPgModel] - PostgreSQL execution model
+ * @param {object} [params.writeConstraints] - Boolean flag to set if record write constraints apply
+ * @returns {Promise<object>} - PostgreSQL execution record that was written to the database
  */
-const _writeExecutionRecord = ({
-  apiRecord,
+const _writeExecutionRecord = async ({
   postgresRecord,
   knex,
   executionPgModel = new ExecutionPgModel(),
-  updatedAt = Date.now(),
-  esClient,
   writeConstraints = true,
-}) => createRejectableTransaction(knex, async (trx) => {
+}) => {
   logger.info(`About to write execution ${postgresRecord.arn} to PostgreSQL`);
-  const [executionPgRecord] = await executionPgModel.upsert(trx, postgresRecord, writeConstraints);
+  const [executionPgRecord] = await executionPgModel.upsert(knex, postgresRecord, writeConstraints);
   logger.info(`Successfully wrote execution ${postgresRecord.arn} to PostgreSQL with cumulus_id ${executionPgRecord.cumulus_id}`);
-  try {
-    await writeExecutionToES({
-      apiRecord,
-      updatedAt,
-      esClient,
-      writeConstraints,
-    });
-    logger.info(`Successfully wrote Elasticsearch record for execution ${apiRecord.arn}`);
-  } catch (error) {
-    logger.info(`Write to Elasticsearch failed, rolling back data store write for execution ${apiRecord.arn}`);
-    throw error;
-  }
   return executionPgRecord;
-});
+};
 
 /**
  * Write execution record to databases and publish SNS message
  *
- * @param {Object} params
- * @param {Object} params.apiRecord - Execution API record to be written
- * @param {Object} params.postgresRecord - Execution PostgreSQL record to be written
- * @param {Object} params.knex - Knex client
- * @param {Object} [params.executionPgModel] - PostgreSQL execution model
- * @param {number} [params.updatedAt] - updatedAt timestamp to use when writing records
- * @param {Object} [params.esClient] - Elasticsearch client
- * @returns {Promise<Object>} - PostgreSQL execution record that was written to the database
+ * @param {object} params
```

Comment:
Nit: We need to get a ticket in to type this properly - the mod to params without type annotations is concerning

Reply:
I have added it to my branch for ticket CUMULUS-3231